tests/network/services.go

```diff
@@ -239,85 +139,113 @@ var _ = SIGDescribe("Services", func() {
-				exposeExistingVMISpec(vmi, subdomain, hostname, selectorLabelKey, selectorLabelValue),
-				console.LoginToFedora)
-		}
-
 		BeforeEach(func() {
-			subdomain := "vmi"
-			hostname := "inbound"
-
-			inboundVMI = createReadyVMIWithMasqueradeBindingAndExposedService(hostname, subdomain)
+			const subdomain = "vmi"
+			const hostname = "inbound"
+
+			inboundVMI = libvmi.NewFedora(append(
+				libvmi.WithMasqueradeNetworking(),
+				libvmi.WithLabel(selectorLabelKey, selectorLabelValue),
+				libvmi.WithSubdomain(subdomain),
+				libvmi.WithHostname(hostname),
+			)...)
+			var err error
+			inboundVMI, err = kubevirt.Client().VirtualMachineInstance(util.NamespaceTestDefault).Create(context.Background(), inboundVMI)
+			Expect(err).ToNot(HaveOccurred())
+
+			inboundVMI = libwait.WaitUntilVMIReady(inboundVMI, console.LoginToFedora)
 			tests.StartTCPServer(inboundVMI, servicePort, console.LoginToFedora)
 		})
 
-		AfterEach(func() {
-			Expect(inboundVMI).NotTo(BeNil(), "the VMI object must exist in order to be deleted.")
-			cleanupVMI(virtClient, inboundVMI)
-		})
-
 		Context("with a service matching the vmi exposed", func() {
-			var jobCleanup func() error
-			var service *k8sv1.Service
-
-			AfterEach(func() {
-				Expect(jobCleanup).NotTo(BeNil(), cleaningK8sv1JobFuncShouldExist)
-				Expect(jobCleanup()).To(Succeed(), cleaningK8sv1JobShouldSucceed)
-				jobCleanup = nil
-			})
-
-			AfterEach(func() {
-				Expect(cleanupService(inboundVMI.GetNamespace(), service.Name)).To(Succeed(), cleaningK8sv1ServiceShouldSucceed)
-			})
-
 			DescribeTable("[Conformance] should be able to reach the vmi based on labels specified on the vmi", func(ipFamily k8sv1.IPFamily) {
+				var service *k8sv1.Service
 				serviceName := "myservice"
-				By("setting up resources to expose the VMI via a service", func() {
-					libnet.SkipWhenClusterNotSupportIPFamily(ipFamily)
-					if ipFamily == k8sv1.IPv6Protocol {
-						serviceName = serviceName + "v6"
-						service = netservice.BuildIPv6Spec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
-					} else {
-						service = netservice.BuildSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
-					}
-
-					_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(context.Background(), service, k8smetav1.CreateOptions{})
-					Expect(err).NotTo(HaveOccurred(), "the k8sv1.Service entity should have been created.")
-				})
 
-				By("checking connectivity the exposed service")
+				libnet.SkipWhenClusterNotSupportIPFamily(ipFamily)
+
+				By("setting up resources to expose the VMI via a service")
+				if ipFamily == k8sv1.IPv6Protocol {
+					serviceName += "v6"
+					service = netservice.BuildIPv6Spec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
+				} else {
+					service = netservice.BuildSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
+				}
+
+				c := kubevirt.Client()
```

Comment:
I prefer the shortform `c`, but everywhere else in this file you have `virtClient`. This inconsistency may perplex the reader.